src/layout/Section.jsx

```diff
@@ -47,6 +51,9 @@ Section.propTypes = {
 	noSeparator: PropTypes.oneOfType([
 		PropTypes.bool,
 		PropTypes.oneOf(Object.keys(VALID_BREAKPOINTS))
+	]),
+	flushUntil: PropTypes.oneOfType([
```

Comment:
Since you got rid of the bool proptype, can't you shorten this to something like:`flushUntil: PropTypes.oneOf(Object.keys(VALID_BREAKPOINTS)),`?